pact/relay/pool.pact

```diff
@@ -104,16 +104,9 @@
   (defcap ACTIVITY ( pool:string bond:string activity:integer)
     @event true)
 
-
-  (defcap ROTATE (bond:string account:string)
-    "Rotation of bond credentials requires auth from originating account."
+  (defcap ROTATE ( bond:string )
     @managed
-    (with-read bonds bond
-      { 'pool := pool, 'account := account }
-      (with-read pools pool
-        { 'token := token:module{fungible-v2} }
-          (enforce-guard (at 'guard (token::details account)))))
-  )
+    (enforce-guard (at 'guard (get-bond bond))))
```

Comment:
I thought it was reasonable to enforce the account guard when rotating the bond guard. Was thinking of bond key as a kind of sub-key that can be used in the relay software. Don't have a strong opinion on it though.

Reply:
That makes sense